packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -230,6 +230,13 @@ interface DatabaseClusterBaseProps {
    * @default false
    */
   readonly iamAuthentication?: boolean;
+
+  /**
+   * Whether to copy tags to the snapshot when a snapshot is created.
+   *
+   * @default: false
+   */
+  readonly copyTagsToSnapshot?: boolean;
```

Comment:
Why did you add this property to `DatabaseClusterBaseProps`? `DatabaseClusterBaseProps` has 2 sub-interfaces, `DatabaseClusterProps` and `DatabaseClusterFromSnapshotProps`. I doubt setting this on a `DatabaseClusterFromSnapshot` makes sense, so this should probably be added in `DatabaseClusterProps`.